Sources/SwiftInspectorVisitors/ProtocolVisitor.swift

```diff
@@ -36,10 +35,17 @@ public final class ProtocolVisitor: SyntaxVisitor {
       return .skipChildren
     }
 
+    let associatedtypeVisitor = AssociatedtypeVisitor()
+    associatedtypeVisitor.walk(node.members)
+
     let typeInheritanceVisitor = TypeInheritanceVisitor()
-    typeInheritanceVisitor.walk(node)
+    if let inheritanceClause = node.inheritanceClause {
```

Comment:
while I was here I decided only to walk the tree if we knew the clauses we were looking for existed.